src/core/index.ts

```diff
@@ -158,3 +158,6 @@ export { getUserColor } from './utils/common';
 
 export { useTheme } from './utils/hooks/useTheme';
 export type { ThemeType } from './utils/hooks/useTheme';
+
+export { Header } from './Header';
```

Comment:
I think in barrel we need to export all the header compnents as they will be used to build Header...
Also, could you put it alphabeticaly ordered? :D

Reply:
Pretty sure the script created that, I can move it and add other stuff.

Reply:
Yeah, script does not handle alphabetical ordering, sadly.